docs/research/matt_research_links.md

```diff
@@ -0,0 +1,57 @@
+# Research Links
+
+Work on gathering some research related to the topics discusses in the last client meeting. Will discuss our findings during the next team meeting (Oct 3rd 2022).
+
+### [Blender/Python Animation aproach (Ashish Jha)](https://www.youtube.com/watch?v=UmhoCumme0o)
+
+- uses python for automated speech to text. 
+- Parses a sentence for vowel phomemes. Sets an angle associated with said vowel phomeme
+<br> **Pros** </br>
+ - very simple
+ - free and open source
+<br>**cons**</br>
+ - Script requires blender.
+ - Wrong tool for the job, blender is a tool made for graphic artists, not plucky CS majors. 
+ - requires addequate knowledge of blender. 
+
+ ### [carykh's aproach to animation](https://www.youtube.com/watch?v=y3B8YqeLCpY)
+*eduoard already researched this so I'm not going to go too in depth.*
+- uses multiple timetables to communicate emotion,pose,topic,phoneme, and paragraph. 
+- uses phomeme library called gentle 
+
+![Chart](2d_stickguy_layout.jpg){:width=250px}
+
+### [Adobe Animate](https://www.youtube.com/watch?v=3pjPJ1YI3GE) 
+*Veronica touched on this but I'll put it here anyways* 
+- Uses a visemes template with images for special phomemes 
+- consolidate as a symbol, do more weird animate stuff
+- use a source audio file text, Animate will do the legwork from there. 
+<br>**Pros**</br>
```

Comment:
Change pros/cos layout

Reply:
Was this resolved ? I think the layout looks okay, not ideal, but good enough